spec/models/glider_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Glider, type: :model do
+  fixtures :all
+
+  describe '#name' do
+    context 'glider name' do # how to improve this?
+      subject(:glider) do
+        Glider.new(glider_type: 'Wave 2', size: 'S', manufacturer: 'Skyjam')
+      end
+
+      it 'returns "Wave 2 | Skyjam"' do # should I use '' and "" the other way around?
+        expect(glider.name).to eq('Wave 2 S | Skyjam')
+      end # should I test also something else?
```

Comment:
When deciding what cases to test you should think about the common ("happy") cases (such as this one) and about edge cases. 

For common cases, usually one or 2 tests are enough.

Edge cases are cases where inputs are special (empty string, nil, 0, negative numbers). Think about if any of the input to your test can realistically be one of these and how your code should react to them. For example in this method, can `glider_type`, `size` or `manufacturer` be `nil` or empty string? If yes, you should write some tests that cover these cases.

Reply:
So I should check if nil returns nil or empty string? Hmmm

Reply:
So looking at the gliders migration, you only have `null: false` for `glider_type`, meaning that only `glider_type` cannot be nil. So you should do this:

Look at all the attributes and think about what `name` should return if they're `nil` or empty string. If it makes no sense that they can be `nil` or empty string, then don't handle that case in the test, but make it impossible (by implementing some validations).

See how writing tests makes you think about these valuable things you might otherwise not ask yourself? ;)